src/Services/ActivationService.cs

```diff
@@ -85,6 +87,18 @@ private async Task StartupAsync()
         // Subscribe to theme changes.
         _themeSelectorService.ThemeChanged += (_, theme) => App.MainWindow.SetRequestedTheme(theme);
         _themeSelectorService.SetRequestedTheme();
+
+        // Disable Hyper-V extension on ARM on first run
+        if (!await _localSettingsService.ReadSettingAsync<bool>(WellKnownSettingsKeys.IsNotFirstRun))
+        {
+            var arch = RuntimeInformation.ProcessArchitecture;

```

Comment:
Note: it may be possible to run x86 or x64 DevHome on [ARM64EC](https://learn.microsoft.com/en-us/windows/arm/arm64ec) devices. We don't install those architectures, but users could force it or architectures we build can change in the future. It would be more correct to check device architecture instead of running process architecture.

Reply:
Changed